### PR TITLE
add workflow documentation

### PR DESCRIPTION
**Pull request by itkg-bfouche**

### CODE REVIEW

File: en/user_guide/workflow.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Workflow
+========
+
+Open orchestrated allows to have a publication workflow.
+It is a series of steps to achieve for publication of a node or a content before it can be displayed on Front Office.
+
+Status
+------
+
+Nodes and contents have status, change it allows to pass the series of steps of publication workflow.
```
Comment (juchi) [inline en/user_guide/workflow.rst]: have a status
changing it
the publication workflow ?

File: en/user_guide/workflow.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+* **Initial status**: fist workflow status (status that a node or a content will have after its creation).
```
Comment (juchi) [inline en/user_guide/workflow.rst]: fist ? :)

File: en/user_guide/workflow.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* **Final status**: latest workflow status (status which allows to display on the Front Office).
+
+The users can define the status which will make up their workflow but it can have only one initial and final status.
+
+Status creation
+~~~~~~~~~~~~~~~
+
+* **Name**(required): status name
+* **Published**: if this status is final
+* **Initial**: if this status is initial
```
Comment (juchi) [inline en/user_guide/workflow.rst:21]: maybe put a note after this list to quickly explain what is "initial" and "final"

File: en/user_guide/workflow.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+-----
+
+Open Orchestra allows to create roles.
+
+* **Name**: role name
+* **Descriptions**: role description in all the languages of the Back Office.
+
+.. image:: ../../images/role_form.png
+
+The roles also allow to create a transition between status. Choose two roles for your transition.
```
Comment (juchi) [inline en/user_guide/workflow.rst]: 1 role and 2 statuses, isn't it ?
Can you try to rephrase with passive voice ?

File: en/user_guide/workflow.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+.. image:: ../../images/role_transition.png
+
+Transitions
+-----------
+
+Every nodes or contents have a status per version and it is transitions creates with the roles which allow to pass from a status to an other.
```
Comment (juchi) [inline en/user_guide/workflow.rst]: Use singular here :
Every node or content has a status [...]
The second part of the sentence is not clear.

File: en/user_guide/workflow.rst [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+.. image:: ../../images/status_transition.png
+
+See also `group management`_ to know how give roles to the users.
```
Comment (juchi) [inline en/user_guide/workflow.rst]: how to give

File: en/user_guide/workflow.rst [hunk 7]
```diff
@@ -0,0 +1,53 @@
+Workflow
+========
+
+Open orchestrated allows to have a publication workflow.
```
Comment (ngilain) [inline en/user_guide/workflow.rst]: Passive form of Open Orchestra ? :D

File: en/user_guide/workflow.rst [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+It is a series of steps to achieve for publication of a node or a content before it can be displayed on Front Office.
+
+Status
+------
+
+Each node or content has a status per version, changing it allows to pass the publication workflow steps.
```
Comment (ngilain) [inline en/user_guide/workflow.rst]: Maybe something like "Each version of a node or a content has a status" is more readable

File: en/user_guide/workflow.rst [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+* A **final** status is the last workflow status (which allows the display on the Front Office).
+
+Status creation
+~~~~~~~~~~~~~~~
+
+* **Name** (required): status name
+* **Published**: if this status is final
+* **Initial**: if this status is initial
+* **Labels**: status label in all the Back Office languages
+* **Color** (required): status color
```
Comment (ngilain) [inline en/user_guide/workflow.rst]: Maybe you can indicate where this color is used

File: en/user_guide/workflow.rst [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+-----
+
+Open Orchestra allows to create roles.
+
+* **Name**: role name
+* **Descriptions**: role description in all the languages of the Back Office.
+
+.. image:: ../../images/role_form.png
+
+See also `group management`_ to know how give roles to the users.
```
Comment (ngilain) [inline en/user_guide/workflow.rst]: how to give

File: en/user_guide/workflow.rst [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+Transitions
+~~~~~~~~~~~
+
+Transitions allow to update a node or a content from a status to another one.
```
Comment (ngilain) [inline en/user_guide/workflow.rst]: I feel like there's too many "allow" in our docs.
Maybe here you could say something like "A transition is a link between two status", and talk about the Workflow graph that results of this. A user guide describes more than graphic interface, it presents concepts behind to help the contributor.

Comment (nicolasThal) [inline en/user_guide/workflow.rst]: Yes in fact, the user guide aims is more to describe the interface, but in this case, I think that a sentence is more ok.

I would propose : `Transition allows users to update a node or a content from one status to another.`

**Comment by nicolasThal** [comment]
:+1: après squash :)

**Comment by juchi** [comment]
:+1:

**Pull request merged**